draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -1263,25 +1198,25 @@ added as an optional SVCB parameter.
       to dedicated sections.
     * Split out a separate HTTPSSVC record for
       the HTTPS use-case.
-    * Remove the explicit SvcRecordType=[01] and instead
+    * Remove the explicit SvcRecordType=0/1 and instead
       make the AliasForm vs ServiceForm be implicit.
       This was based on feedback recommending against
       subtyping RRTYPE.
     * Remove one optimization.
-    
+
 * draft-nygren-httpbis-httpssvc-03
-    * Change redirect type for HSTS-style behavior 
+    * Change redirect type for HSTS-style behavior
       from 302 to 307 to reduce ambiguities.
-    
+
 * draft-nygren-httpbis-httpssvc-02
     * Remove the redundant length fields from the wire format.
-    * Define a SvcDomainName of "." for SvcRecordType=1 
+    * Define a SvcDomainName of "." for SvcRecordType=1
       as being the HTTPSSVC RRNAME.
     * Replace "hq" with "h3".
 
 * draft-nygren-httpbis-httpssvc-01
     * Fixes of record name.  Replace references to "HTTPSVC" with "HTTPSSVC".
-    
+
 * draft-nygren-httpbis-httpssvc-00
     * Initial version
-    
+
```

Comment:
There was only one instance of "a SVCB", so I went with the majority.

Reply:
I think this may be because it used to be "an ALTSVC" and then "an HTTPS" so when I did the query-replace I didn't fix/replace.  This can get cleaned up later.